### PR TITLE
Make kubemark scripts fail fast

### PR DESCRIPTION
**Pull request by shyamjvs**
Fixes https://github.com/kubernetes/kubernetes/issues/46601

/cc @wojtek-t @gmarek

### CODE REVIEW

File: test/kubemark/resources/start-kubemark-master.sh [hunk 1]
```diff
@@ -493,9 +496,17 @@ start-kubemaster-component "kube-controller-manager"
...
 
-# Wait till apiserver is working fine.
+# Wait till apiserver is working fine or timeout.
+echo -n "Waiting for apiserver to be healthy"
+start=$(date +%s)
 until [ "$(curl 127.0.0.1:8080/healthz 2> /dev/null)" == "ok" ]; do
+	echo -n "."
 	sleep 1
+	now=$(date +%s)
+	if [ $((now - start)) -gt 180 ]; then
```
Comment (wojtek-t) [inline test/kubemark/resources/start-kubemark-master.sh]: I would make the timeout bigger. Maybe 300 or even more?
180 sounds too aggresive to me.

File: test/kubemark/resources/start-kubemark-master.sh [hunk 2]
```diff
@@ -93,7 +97,6 @@ function safe-format-and-mount() {
 		mkfs.ext4 -F "${device}"
 	fi
 
-	mkdir -p "${mountpoint}"
```
Comment (wojtek-t) [inline test/kubemark/resources/start-kubemark-master.sh]: Why did you remove this?

Comment (shyamjvs) [inline test/kubemark/resources/start-kubemark-master.sh]: Because it is already created before calling this function, inside mount-pd.

File: test/kubemark/resources/start-kubemark-master.sh (hunk 1 above)
Comment (shyamjvs) [inline test/kubemark/resources/start-kubemark-master.sh]: Done. This is just waiting for apiserver to be healthy, it should've started pretty quickly above. Even restarting it 3-4 times shouldn't take too much time, so maybe 180 was enough? But 300 sgtm.

**Comment by wojtek-t** [comment]
@k8s-bot pull-kubernetes-e2e-kops-aws test this
